### ISSUE TITLE
Strange plotting behavior in the plot_ppc function

### ISSUE THREAD

**Issue by RaulPL** [arviz-devs/arviz#751]
**Describe the bug**
I'm trying to plot the posterior predictive of the rugby dataset against the observed data but I get a strange plot that looks like this:

![image](https://user-images.githubusercontent.com/3116652/61176682-9aa3b280-a58a-11e9-826e-a45a2d4c4359.png)

**To Reproduce**
I use the following code to generate the image:
```
import arviz as az
import matplotlib.pyplot as plt
rugby_data = az.load_arviz_data("rugby")
az.plot_ppc(rugby_data, alpha=0.1, figsize=(15, 6), textsize=14, kind='density')
```

**Expected behavior**
I think I should get a a plot similar to this one:

```
fig, axes = plt.subplots(1, 2, figsize=(15, 4), sharex=True)
az.plot_dist(rugby_data.observed_data.away_points.values, ax=axes[0], label='observed_data')
az.plot_dist(rugby_data.observed_data.home_points.values, ax=axes[1], label='observed_data')

az.plot_dist(rugby_data.posterior_predictive.away_points.values.flatten(), ax=axes[0], color='red', label='posterior')
az.plot_dist(rugby_data.posterior_predictive.home_points.values.flatten(), ax=axes[1], color='red', label='posterior')

axes[0].set_title('observed away_points')
axes[1].set_title('posterior away_points')
```

![image](https://user-images.githubusercontent.com/3116652/61176807-fc651c00-a58c-11e9-83f7-b3aa9f24126d.png)

The problem might be  due to the number of bins that are been assigned to the posterior predictive histogram, the bars seem to be really thin. 

**Additional context**
In my last fig the observed and posterior plots are different (kde and bars) because in the rugby dataset we have different types (floats and ints).

**Comment by RaulPL** [comment]
I think I found the source of the problem, in this [line](https://github.com/arviz-devs/arviz/blob/master/arviz/plots/ppcplot.py#L328) the number of bins (nbins) is a function of the number of samples in the chains (which is usually big), I think it should be a function of the number of observations (obs_vals)  like in the rest of the function. This change fixed the previous plot 
```
nbins = round(len(obs_vals) ** 0.5)
```
![image](https://user-images.githubusercontent.com/3116652/61177344-ae561580-a598-11e9-87b4-46cce03e7367.png)

**Comment by OriolAbril** [comment]
@RaulPL fixed this in #752!
